src/components/admin/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -51,14 +47,16 @@ export const ConfirmationModal: React.FC<ConfirmationModalProps> = ({
 
   return (
     <Modal show={isShown} onHide={hide}>
-      <div className="ConfirmationModal">
+      <div className="flex flex-col items-center">
```

Comment:
To help us visually navigate DOM with browser inspect, but also not interfere with styles, first prop can be made a BEM style value named `data-bem=`

```suggestion
      <div data-bem="ConfirmationModal" className="flex flex-col items-center">
```

This comment is valid for all instances like this or any other instances we might have bare elements and might be helpful to annotate as such.

Reply:
[1ae2f8b](https://github.com/sparkletown/sparkle/pull/2944/commits/1ae2f8b185da5056b37e5478d84f29d2daf6cce8)